can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -377,8 +371,17 @@ def send(self, msg, timeout=None):
             message.NetworkID = self.channels[0]
         else:
             raise ValueError("msg.channel must be set when using multiple channels.")
+        return message
+
+    def send(self, msgs, timeout=None):
```

Comment:
Why is this overridden in here?

Reply:
Ahh. I'd just add a comment.

```suggestion
    def send(self, msgs, timeout=None):
        # we always pass the entire sequence as a whole
```